js/src/layers/MarkerCluster.js

```diff
@@ -34,6 +34,30 @@ export class LeafletMarkerClusterView extends layer.LeafletLayerView {
       },
       this
     );
+    this.listenTo(
+      this.model,
+      'change:opacity',
+      function () {
+        if (this.model.get('visible')) {
+          document.querySelector('.test-icon').style.opacity =
```

Comment:
This would not work if you have multiple marker clusters on the map and want to hide one and not the other. Hiding one would hide the other and that's unfortunately not what you want.